eZ/Bundle/EzPublishCoreBundle/Resources/config/routing/content.yml

```diff
@@ -0,0 +1,3 @@
+_ezplatform_download:
+    path: /content/download/{contentId}/{fieldIdentifier}/{downloadName}
+    defaults: {_controller: ezpublish.controller.content.download:downloadBinaryFileAction}
```

Comment:
You'll probably want to add this route to [`ezpublish.default_router.legacy_aware_routes`](https://github.com/ezsystems/LegacyBridge/blob/master/bundle/Resources/config/services.yml#L6) in legacy bridge. Or maybe you prefer legacy to continue delivering binary files by itself.